### ISSUE TITLE
Putting the FAQ page within a CMS type page in the back end.

### ISSUE THREAD

**Issue by robgpeak** [ChrisASearles/CompLinks#168]
@cgladue I think it would be helpful if the FAQ page was easily editable from the back end.  Meaning, it would be great to be able to change and add content to the page from the admin dashboard.

**Comment by robgpeak** [comment]
In speaking with some industry execs in the loyalty rewards space, they say the single biggest issue they receive from end users is an inquiry about missing points/transaction.  Chad is already working on a system where a CL staff member can look up "orphan transactions" (transactions which we've been paid a commission, but for one reason or another, they we not tagged to a particular end user's account.

To that end, and important FAQ will be, "What if I made a purchase, but did not receive my points?  Answer: 

It is important to understand that different merchants take a different amount of time to report back to us the number of points to issue to your account.  Some big name brand store can report within hours, others take days, and some smaller merchants can take up to a month.

First thing to do is check your Shopping Trips page in the My Account section of our website (you must be logged into your account), to see if there was a "Trip ID" recorded.  If there WAS a trip recorded, you should be all set, and might just have to wait until the merchant reports the transaction.

If there was no trip recorded, please send and email to support@complinks.co with the Order ID and the Date of the transaction.  Both of these bits of information should be located in the confirmation email you would receive from the merchant directly after making a purchase on their site.

Another important note is that we can only issue points for shopping trips that are started from a link on our site, AND are completed online.  Purchases that are made by placing an order over the phone to the merchant will not earn points, even if you went to the merchants site from our, as there is no way to track those transactions at this time.

**Comment by cgladue** [comment]
i dont know if its there already but another use case might also be that if they started a trip from our site, but then somehow used another app or another site that overrode our referral, meaning only the latest redirect to the advertiser store will be in effect, so if they start a trip from our site then go to ebates and decide to use their link it will override our trip.

**Comment by robgpeak** [comment]
Yeah, this is another reason why it would be good for us to have the ability to edit this page easily and regularly.  I am sure once we get the site up and running, we will encounter many issues that we would want to update out FAQ for...

**Comment by ChrisASearles** [comment]
This should be handled by the support software. I still have to look into it but if it's not, I'd recommend a switch to something that allows more flexibility for support staff (like Zendesk). I'll get back to this once I look into the support side.

**Comment by robgpeak** [comment]
Chris, I know support software handles this sort of thing, (knowledge bases, etc.), but for now it does not need to be that complicated. I think the easiest solution is to make the page behave just like if it was database driven page, and Morgan can edit it just like she edits store descriptions and the "conditions" sections.  This is something Chad could do in his sleep:)  So @cgladue next time you're taking a nap, you can bust this out.  lol

**Comment by ChrisASearles** [comment]
Hey @robgpeak and @morganpeak Groove has this functionality built in and it's very easy to use. I spent 10 minutes setting this up and you guys can add as necessary. You can check out what I did here: https://complinks.groovehq.com/help_center

I published your FAQ in it's entirety as an article, and began creating some other categories and published each question as it's own article within specific categories as well. Let me know if you have any problems working with this.

I just wanted to take the 10 minutes to set this up using the system you're already paying for so you could see how it worked and then decide if you want to still build something custom out.

I configured this to work on the https://help.complinks.co CNAME so if you want to use this we need to ask @cgladue to add that record and then we can figure out how you want to link to it within the site.

**Comment by morganpeak** [comment]
I like it a lot. Like you said, since we're paying for it and they offer this type of system we might as well use it. Let's have @cgladue add the record and I'll talk to @robgpeak about where to put it within the site

**Comment by robgpeak** [comment]
@cgladue please set up a CNAME in cloudflare for "help" and then on the front end of shop.complinks.co link the FAQ links to help.complinks.co. Question though, what happens with the sub domains help sections...?  I guess I will answer my own question and say let's let them all go to the same generic FAQ page we have now until we grow and see if it becomes an issue.  The potential issue would be people asking casino specific related questions, but then again they can just send us a message.

So it short, let's do this the way Chris built it and described it.

**Comment by cgladue** [comment]
it would have to be a generic help page it wouldnt be different per partner.



Chad Gladue
*DevNull Technology, LLC*

**Comment by robgpeak** [comment]
Yes it will be generic. We will only be answering FAQs as it relates to our portion of the program, and if they have casino specific questions, they (the user) will just have to send us a message.

So let's set up the CNAME on cloudflare to point to that groove page Chris quoted, and then Chris can set up the links on the front end.

**Comment by robgpeak** [comment]
@cgladue Please go to https://help.complinks.co/ and see the SSL error message and let me know if there is anything I can/need to to to remedy the situation?

**Comment by cgladue** [comment]
seems like it is because the cert on cloudflare is the *.complinks.co cert but help.complinks.co actually redirects to help.groovehq.com so therefore that does not match the wildcard cert on cloudflare, we may be able to turn off cloudflare for the help subdomain and not have it use cloudflare, give me a second.

**Comment by cgladue** [comment]
See Response below from groove, seems like for the time being will need to only use HTTP and not HTTPS for the help KB page.

=====================

Hi Chad, 

Sorry for the confusion here, we unfortunately don't support SSL on custom domains right now. This feature is planned for our next version of the KB, however this is still at least a few months out on our roadmap. 

I've added you to our list for follow up and we'll be sure to reach out as soon as the new KB gets released. 

Cheers,
Lesley

**Comment by robgpeak** [comment]
OK.  That should not be a problem I suppose.  Do I need to do anything?

**Comment by cgladue** [comment]
just where ever help.complinks.co is referenced it should be changed to http and not https, other then that i dont think you need to do anything right now.

**Comment by robgpeak** [comment]
Something is a little screwy because the actual page on their site which is ours is https://complinks.groovehq.com/help_center and it is https, but right now help.complinks.co in cloudflare is pointing to just complinks.groovehq.com without the rest of the URL (/help_center), so that is not the right page anyway...
